### ISSUE TITLE
Warping to moc_fild04 crashes Map Server

### ISSUE THREAD

**Issue by Edduaardo** [rathena/rathena#3740]
* **rAthena Hash**: 348acd187

* **Client Date**: 20170517a RagRE.

* **Server Mode**: Renewal.

* **Description of Issue**: 
  * Result: Map Server receives crash signal when a player tries to warp to moc_fild04 (Sograt Desert), map_getmapflag returns a invalid ID -1.
  * Expected Result: Teleporting to this map without Map Server crashing.
  * How to Reproduce: Just issue the command @ warp moc_fild04
  * Official Information:
  

* **Modifications that may affect results**:

**Comment by anacondaqq** [comment]
As far as I know, moc_fild04 is a part of an old episode, and in the current episodes are removed. That means the bug happens when trying to access a map which does not exist in map-cache or something like that. Hmm, interesting.

**Comment by Edduaardo** [comment]
You are right, it looks like it's some sort of special map, probably the default "Map not found" error handler (I think it's called like this) is missing for this map. The crash disconects everyone from the server

**Comment by aleos89** [comment]
This was fixed in 505a0cd.
